### DISCUSSION TITLE
Failed to create database

### DISCUSSION THREAD

**Discussion by Hypnosekroete** [kimai/kimai#4081] [On-Premise: Installation & Updates] [answered]
Hi everybody,
i want to try kimai.
On the installation i receive the following error (Step: bin/console kimai:install -n):
_[ERROR] Failed to create database: Connection does not contain a 'path' or     
         'dbname' parameter and cannot be created._       
I did the installation with the guide for fresh ubuntu 22.04.
The .env file was modified (Did the Setup with the Maria DB)
Database Url in my .env file

_DATABASE_URL=mysql://Kimai:mypw@127.0.0.1:3306/database?charset=utf8&serverVersion=mariadb-10.6.12_

Has anyone an hint or an idea for the error?

**Comment by kevinpapst** [comment, marked as answer]
I guess you have character in the password, which need to be url encoded. 
See https://www.kimai.org/documentation/installation.html#malformed-parameter-url
Safest way is to use a long username/password that only consists of alpha-numeric character.

**Reply by Hypnosekroete** [reply]
Thank you very much, that was the problem.
To be honest, i went crazy with that error message. (Proofing the Database URL many times)
